config/babylon.go

```diff
@@ -33,7 +33,7 @@ func DefaultBBNConfig() BBNConfig {
 		GRPCAddr:       dc.GRPCAddr,
 		AccountPrefix:  dc.AccountPrefix,
 		KeyringBackend: dc.KeyringBackend,
-		GasAdjustment:  dc.GasAdjustment,
+		GasAdjustment:  20, // Increased from default to ensure sufficient gas
```

Comment:
hmm default is `1.2` increasing it by 20x seems a bit harsh. Maybe lets set default to `5` ?